src/bptl/work_units/email/tasks.py

```diff
@@ -0,0 +1,89 @@
+from django.conf import settings
+from django.core.mail import send_mail
+from django.template.loader import get_template
+from django.utils.translation import ugettext_lazy as _
+
+from zgw_consumers.api_models.base import factory
+
+from bptl.tasks.base import WorkUnit
+from bptl.tasks.registry import register
+
+from .serializers import SendEmailSerializer, ValidTemplateChoices
+
+__all__ = ["SendEmail"]
+
+
+@register
+class SendEmailTask(WorkUnit):
+    """This task sends an email to ``receiver`` signed by ``sender``.
+
+    **Required process variables**
+
+    * ``sender``: JSON with email and name of sender.
+        .. code-block:: json
+                {
+                    "email": "kees@example.com",
+                    "name": "Kees Koos"
+                }
+
+    * ``receiver``: JSON with email and name of receiver.
+        .. code-block:: json
+                {
+                    "email": "jan@example.com",
+                    "name": "Jan Janssen"
+                }
+
+    * ``email``: JSON with email variables:
+        .. code-block:: json
+                {
+                    "subject": "This is an example subject.",
+                    "content": "This is an example body."
+                }
+
+    * ``template``: string with template name. Choices:
+        .. code-block:: list
+                [
+                    generiek, accordering, advies,
+                ]
+
+    **Optional process variables**
+
+    * ``kownslFrontendUrl``: string with full URL for end users to submit their review.
+                Valid choices are currently: generic/approval/advice.
+    * ``reminder``: boolean.
+    * ``deadline``: datetime: "2020-04-20 16:20:00".
```

Comment:
Can we make the examples timezone-aware?

```suggestion
    * ``deadline``: datetime: "2020-04-20T16:20:00+00:00".
```